src/Ryujinx/Input/GTK3/GTK3MappingHelper.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Ryujinx.Input.GTK3
 {
-    public static class GTK3MappingHelper
+    public static class Gtk3MappingHelper
```

Comment:
```suggestion
    public static class GTK3MappingHelper
```
Same as other comment